gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -193,6 +195,12 @@ fluid.defaults("gpii.flowManager.local", {
         },
         userErrors: {
             type: "gpii.userErrors"
+        },
+        gpiiRevisionRequester: {
```

Comment:
`revisionRequester` should be a subcomponent of `repositorySolutionsLoader` as that's where it's needed.

It would be easier to search if "gpiiRevisionRequester" could be renamed to be the same as the component name "revisionRequester".